### PR TITLE
feat: Add function to get elapsed time of playing audio on VoiceClient

### PR DESCRIPTION
**Pull request by felix920506**
## Summary

Adds a function to get the elapsed seconds of currently playing audio file

## Information

- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [x] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [x] I have updated the changelog to include these changes.

### CODE REVIEW

**Comment by JustaSqu1d** [comment]
Seconds seem really inflexible, what if we returned a `datetime.timedelta` object instead?

**Comment by felix920506** [comment]
Sounds reasonable

**Comment by felix920506** [comment]
Can I have a review on this?

File: discord/player.py [hunk 1]
```diff
@@ -755,6 +756,8 @@ def _do_run(self) -> None:
                 self._start = time.perf_counter()
 
             self.loops += 1
+            self._played_frames += 1
```
Comment (Dorukyum) [inline discord/player.py:759]: Couldn't we utilise self.loops instead of creating another counter?

**Comment by felix920506** [comment]
Changed to use loops + an offset to keep track of when loops gets reset

File: discord/voice_client.py [hunk 2]
```diff
@@ -988,3 +989,10 @@ def send_audio_packet(self, data: bytes, *, encode: bool = True) -> None:
             )
 
         self.checked_add("timestamp", opus.Encoder.SAMPLES_PER_FRAME, 4294967295)
+
+    def played_time(self) -> datetime.timedelta:
+        """Gets the elapsed time of the playing audio. Returns 0 if not playing anything."""
```
Comment (Dorukyum) [inline discord/voice_client.py:994]:
```suggestion
        """Returns the elapsed time of the playing audio."""
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -22,6 +22,8 @@ These changes are available on the `master` branch, but have not yet been releas
   `tags`. ([#2520](https://github.com/Pycord-Development/pycord/pull/2520))
 - Added `Member.guild_banner` and `Member.display_banner` properties.
   ([#2556](https://github.com/Pycord-Development/pycord/pull/2556))
+- Added `played_time` function to `VoiceClient` to get elapsed time of the playing audio
+  file. ([#2587](https://github.com/Pycord-Development/pycord/pull/2587/))
```
Comment (Dorukyum) [inline CHANGELOG.md:26]:
```suggestion
- Added `elapsed` method to `VoiceClient`.
  ([#2587](https://github.com/Pycord-Development/pycord/pull/2587/))
```

File: discord/voice_client.py [hunk 4]
```diff
@@ -988,3 +989,10 @@ def send_audio_packet(self, data: bytes, *, encode: bool = True) -> None:
             )
 
         self.checked_add("timestamp", opus.Encoder.SAMPLES_PER_FRAME, 4294967295)
+
+    def played_time(self) -> datetime.timedelta:
```
Comment (Dorukyum) [inline discord/voice_client.py:993]:
```suggestion
    def elapsed(self) -> datetime.timedelta:
```

File: discord/voice_client.py [hunk 5]
```diff
@@ -988,3 +989,10 @@ def send_audio_packet(self, data: bytes, *, encode: bool = True) -> None:
...
+        """Returns the elapsed time of the playing audio."""
+        if self._player:
+            return datetime.timedelta(milliseconds=self._player.played_frames() * 20)
+        else:
+            return datetime.timedelta()
```
Comment (Dorukyum) [inline discord/voice_client.py:998]:
```suggestion
        return datetime.timedelta()
```

**Review by Dorukyum — CHANGES_REQUESTED** [review]
I find `client.elapsed()` to be clearer and simpler, what do you think?

**Comment by felix920506** [comment]
I don't have any preference between client.elapsed() or client.played_time()
If you think client.elapsed() is better then let's use that

**Review by Dorukyum — APPROVED** [review]

**Review by Lulalaby — APPROVED** [review]

**Pull request merged**